config/webpack.dev.js

```diff
@@ -31,10 +31,11 @@ const PORT = process.env.PORT || localConfig.devServerPort;
 const EXPRESS_PORT = process.env.EXPRESS_PORT || 9001;
 
 const plugins = [
-  new webpack.NoEmitOnErrorsPlugin(),
```

Comment:
What's the reason for removing these other plugins?

Reply:
VScode had a line crossed through them saying they were deprecated. Cant find anything to support that. I will add them back in.

Reply:
![image](https://user-images.githubusercontent.com/11218376/94065915-bffc9500-fdb9-11ea-95e7-919736c9a11a.png)

Reply:
oh interesting ok. 👍